cachi2/core/models/sbom.py

```diff
@@ -135,7 +139,7 @@ def to_spdx(self, doc_namespace: str) -> "SPDXSbom":
             for prop in component.properties:
                 annotations.append(
                     SPDXPackageAnnotation(
-                        annotator="Tool:cachi2:jsonencoded",
+                        annotator="Tool: cachi2:jsonencoded",
```

Comment:
Out of curiosity, is any of `Tool` and `jsonencoded` somewhat common/used in the SPDX ecosystem?